packages/ci/test/commands/ci/index.test.ts

```diff
@@ -35,31 +35,57 @@ describe('ci', () => {
     })
 
     test
-    .stdout()
-    .nock('https://api.heroku.com', api => {
-      api.get(`/pipelines?eq[name]=${pipeline.name}`)
-      .reply(200, [
-        {
-          id: pipeline.id,
-          name: pipeline.name
+      .stdout()
+      .nock('https://api.heroku.com', api => {
+        api.get(`/pipelines?eq[name]=${pipeline.name}`)
+          .reply(200, [
+            {
+              id: pipeline.id,
+              name: pipeline.name
+            }
+          ])
+
+        api.get(`/pipelines/${pipeline.id}/test-runs`)
+          .reply(200, testRuns)
+      })
+      .command(['ci', `--pipeline=${pipeline.name}`])
+      .it('it shows the latest 15 test runs', ({stdout}) => {
```

Comment:
@talbright no need to say `it` again.

Reply:
I noticed that all the test files in this package have an `it` for every `describe`, and from appearances it looks necessary to be able to grab output.  Can you point me to an example of what approach you prefer?

Reply:
@talbright here's an example:

`it('shows the lastest 15 test runs', ({stdout}) => {`

Reply:
ah I see it now! I didn't notice the `it` stuttering.